assets/translations/en.json

```diff
@@ -58,6 +58,8 @@
   "addHashtag(s)": "Add Hashtag(s)",
   "hashtagshelpotherusersfindyourwagereasilyandquickly": "Hashtags help other users find your\nwager easily and quickly",
   "recentTransactions": "Recent Transactions",
+  "accountSettings":"ACCOUNT SETTINGS",
```

Comment:
don't add text here in all CAP, use .toUpperCase() in the text widget to capitalize a text